data/lib/workspace/generate/descriptor.py

```
@@ -48,8 +52,8 @@ def create(
         timestamp = datetime.datetime.now().timestamp()
         app_path = PROJECT_ROOT / "pubspec.yaml"
         with open(app_path, "r", encoding="utf-8") as f:
-            data = yaml.load(f, yaml.CLoader)
-        app_version = data["version"]
+            pubspec = yaml.load(f, yaml.CLoader)
```

Ruff (0.15.15)

[error] Probable use of unsafe loader `CLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)